examples/7nodes/init.sh

```diff
@@ -19,13 +19,26 @@ function createPermissionedNodesJson(){
     i=$(( ${nodes} + 1))
 
     permFile=./permissioned-nodes-${nodes}.json
-    rm -f ${permFile}
-    if [[ "$nodes" -ne 7 ]] ; then
+    creFile=true
```

Comment:
Not sure if I'm missing something here, but do we need this code in init.sh?
It seems to be duplication of what is covered in the *-init.sh scripts for the specific consensus versions.

Reply:
Agree Satpal. I was trying to use `source` in the bash scripts and somehow it was not working in mac. Ended up duplicating the code. The idea of the changes was to ensure that if some one calls any of the init scripts directly instead of using `init.sh` it should still work. Have refactored the code and removed the duplicity